dataplane/saiserver/routing_test.go

```diff
@@ -226,15 +222,22 @@ func TestCreateNextHopGroupMember(t *testing.T) {
 			}},
 		},
 		wantAttr: &saipb.NextHopGroupMemberAttribute{
-			NextHopGroupId: proto.Uint64(1),
-			NextHopId:      proto.Uint64(2),
+			NextHopGroupId: proto.Uint64(2),
+			NextHopId:      proto.Uint64(3),
 			Weight:         proto.Uint32(3),
 		},
 	}}
 	for _, tt := range tests {
 		t.Run(tt.desc, func(t *testing.T) {
 			dplane := &fakeSwitchDataplane{}
 			c, mgr, stopFn := newTestNextHopGroup(t, dplane)
+
```

Comment:
Can we add some comments on these numbers: 10 and 3?

Reply:
they are just random numbers i picked